lib/mhv_ac/upgrade_form.rb

```diff
@@ -1,19 +1,13 @@
 # frozen_string_literal: true
 
 require 'common/models/form'
-require 'common/models/attribute_types/httpdate'
 
 module MHVAC
   class UpgradeForm < Common::Form
     include ActiveModel::Validations
 
     attribute :user_id, Integer
-    attribute :form_signed_date_time, Common::HTTPDate
-    attribute :form_upgrade_manual_date, Common::HTTPDate
-    attribute :form_upgrade_online_date, Common::HTTPDate
```

Comment:
Are we sure that the MHV api no longer requires us to provide these?

Reply:
I think they may still requirement. From a local test, I'm getting a "data integrity issue" from the upgrade request. Should we just use the current date or work with MHV to remove the requirement?

Reply:
I've also gotten an "unknown application error" from trying to upgrade another account (Advanced).

Reply:
They should remove the requirement. It doesn't make sense for us to say the user signed a form or accepted T&C if they didn't actually do so.

Reply:
@davidsbao thoughts?